mailbox/api/src/main/java/org/apache/james/mailbox/exception/HasEmptyMailboxNameInHierarchyException.java

```diff
@@ -20,16 +20,7 @@
 package org.apache.james.mailbox.exception;
 
 public class HasEmptyMailboxNameInHierarchyException extends MailboxNameException {
-    public HasEmptyMailboxNameInHierarchyException() {
```

Comment:
(idem)

Reply:
Because it is unused, and easy to add if we need it?